src/views/ConsentContinuationOut.vue

```diff
@@ -74,8 +74,9 @@
                       <strong>Detail</strong>
                     </v-col>
                     <v-col cols="12" sm="9">
-                      <p class="detail-text">{{defaultComment}}</p>
+                      <strong class="grey-text">{{defaultComment}}</strong>
```

Comment:
I'm not sure that the strong element can have classes... More typical would be a paragraph element with a `font-weight-bold` class.

Ref: https://vuetifyjs.com/en/styles/text-and-typography/#font-emphasis